eng/pipelines/runtime-linker-tests.yml

```diff
@@ -0,0 +1,33 @@
+# The purpose of this pipeline is to exercise local developer workflow in the consolidated
+# runtime repo. In particular, it is supposed to run the root "build" script just like any
+# normal developer normally would and monitor regressions w.r.t. this fundamental scenario.
+
+trigger: none
+
+pr:
+  branches:
+    include:
+    - master
+
+jobs:
+#
+# Checkout repository
+#
+- template: /eng/pipelines/common/checkout-job.yml
+
+#
+# Build Release config vertical for Windows, Linux, Linux musl and OSX
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    platforms:
+    - Windows_NT_x64
+    - OSX_x64
+    - Linux_x64
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: Runtime_Release
+      buildArgs: -s clr+libs -c release /p:LinkerTests=true
```

Comment:
Use $(_BuildConfig) so that we don’t need to update multiple places and so that this is configurable based on the jobParams.

Reply:
I can do that, but as explained above, we generally will always want to build release on this pipeline due to the nature of the linker.